codefresh/pipeline-creator/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
+
+export PROJECT="${INPUT_PROJECT}"
+export GITHUB_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+git clone "${INPUT_REPO_URL}"
+cd "${INPUT_REPO_NAME}" || exit 1
+git checkout "${INPUT_VERSION}"
+
+codefresh auth create-context context --api-key "$INPUT_CF_API_KEY"
```

Comment:
> `set -x` enables a mode of the shell where all executed commands are printed to the terminal. 

This will echo the codefresh API key on the command line. Please delete your codefresh API key and start again.

Reply:
it's not compromised since I did not run the action itself (ran the scripts only).